flambe/field/text.py

```diff
@@ -65,6 +65,46 @@ def get_embeddings(
     return model
 
 
+class EmbeddingsInformation(NamedTuple):
+    """
+    Information about an embedding model.
+
+    Parameters
+    ----------
+    embeddings : str
+        Path to pretrained embeddings or the embedding name
+        in case format is gensim.
+    embeddings_format : str, optional
+        The format of the input embeddings, should be one of:
+        'glove', 'word2vec', 'fasttext' or 'gensim'. The latter can
+        be used to download embeddings hosted on gensim on the fly.
+        See https://github.com/RaRe-Technologies/gensim-data
+        for the list of available embedding aliases.
+    embeddings_binary : bool, optional
+        Whether the input embeddings are provided in binary format,
+        by default False
+    setup_all_embeddings: bool
+        Controls if all words from the optional provided
+        embeddings will be added to the vocabulary and to the
+        embedding matrix. Defaults to False.
+    unk_init_all : bool, optional
+        If True, every token not provided in the input embeddings is
+        given a random embedding from a normal distribution.
+        Otherwise, all of them map to the '<unk>' token.
+    drop_unknown: bool
+        Whether to drop tokens that don't have embeddings
+        associated. Defaults to True.
+        Important: this flag will only work when using embeddings.
+
+    """
+    embeddings: str
+    embeddings_format: str = 'gensim'
+    embeddings_binary: bool = False
+    setup_all_embeddings: bool = False
+    unk_init_all: bool = False
```

Comment:
this is not necessary embedding related. Neither is drop_unknown

Reply:
iirrc and according to the documentation, this parameters are only used in case embeddings are provided. Do you have an example where they are not used with embeddings?

Reply:
`unk_init_all` is for for the vocab that is not in the embeddings.

Reply:
well for drop_unkown you can drop things that are not in the vocab, and you may have created the vocab from the data not the embeddings file

Reply:
and `drop_unknown` just works now when embeddings are passed. Maybe we can generalize this one to work also for non embedding TextFields

Reply:
> well for drop_unkown you can drop things that are not in the vocab, and you may have created the vocab from the data not the embeddings file

Yes lets generalize this concept